client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -45,19 +45,19 @@ const TitanSetUpThankYou = ( {
 }: TitanSetUpThankYouProps ) => {
 	const currentRoute = useSelector( getCurrentRoute );
 	const selectedSite = useSelector( getSelectedSite );
-	const selectedSiteSlug = selectedSite?.slug ?? ( isDomainOnlySite ? domainName : null );
+	const selectedSiteSlug = isDomainOnlySite ? domainName : selectedSite?.slug;
```

Comment:
Don't we brake something here? Since the condition works different way now.
Do we indeed not have `selectedSite?.slug` when `isDomainOnlySite === true`?